packages/@aws-cdk/aws-ec2/lib/instance-types.ts

```diff
@@ -193,6 +193,16 @@ export enum InstanceClass {
    */
   COMPUTE5_NVME_DRIVE = 'c5d',
 
+  /**
+   * Compute optimized instances based on AMD EPYC, 5th generation.
+   */
+  COMPUTE5_AMD = 'c5a',
+
+  /**
+   * Compute optimized instances based on AMD EPYC, 5th generation
+   */
+  C5A = 'c5a',
```

Comment:
Why the AMD suffix? Can we keep the same convention as for c5?

```
  /**
   * Compute optimized instances, 5th generation
   */
  COMPUTE5 = 'c5',

  /**
   * Compute optimized instances, 5th generation
   */
  C5 = 'c5',
```

Reply:
I use the same convention as m5a
```typescript
    /**
     * Standard instances based on AMD EPYC, 5th generation.
     */
    STANDARD5_AMD = "m5a",
    /**
     * Standard instances based on AMD EPYC, 5th generation
     */
    M5A = "m5a",
```